css/team.css

```diff
@@ -0,0 +1,345 @@
+/*
+===================================
+===================================
+Our Team Styles
+===================================
+===================================
+*/
+
+/*
+===================================
+Member Card Component
+===================================
+*/
+/* member card style */
+.member {
+	background-color: var(--color-teal);
+	color: var(--text-color-light);
+	padding: var(--spacing-sm);
+	border-radius: var(--border-radius);
+	max-width: 654px;
+}
+/* img styling*/
+.member .img {
+	max-width: 240px;
+}
+.member img {
+	border-radius: var(--border-radius);
+}
+/* text */
+.member h2 {
+	font-size: 28px;
+	font-weight: 700;
+	line-height: 32px;
+}
+
+.member .role {
+	font-size: 24px;
+	font-weight: 500;
+	line-height: 28px;
+}
+.member p {
+	font-size: 16px;
+	font-weight: 400;
+	line-height: 20px;
+}
+
+/* member card layout */
+.member,
+.member .text,
+.member .bio,
+.member-cat {
+	display: flex;
+}
+
+/* similar gaps */
+.member,
+.member .text,
+.member-cat {
+	gap: var(--spacing-sm);
+}
+
+.member .bio {
+	gap: 10px;
+}
+
+/* text container and bio  */
+.member .text,
+.member .bio {
+	flex-direction: column;
+}
+.member .text {
+	justify-content: space-between;
+}
+
+/* team modal styles */
+.team-modal .member {
+	background: none;
+	width: 100%;
+	max-width: initial;
+	color: var(--text-color-dark);
+	padding: var(--spacing-sm) 0;
+}
+
+.team-modal .img {
+	max-width: 140px;
+}
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+
+.hero {
+	background: url("/assets/images/pages/team/hero-team.png"),
+		lightgray 50% / cover no-repeat;
+	display: flex;
+}
+
+/* flex layout */
+.hero .wrapper {
+	display: flex;
+	padding: var(--spacing-lg);
+	gap: var(--spacing-xxl);
+}
+
+.hero-img,
+.hero-text-container {
+	width: 50%;
+}
+
+.hero-img {
+	max-width: 435px;
+}
+.hero-text-container {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Team Members
+===================================
+*/
+
+/* layout */
+.team-members {
+	display: flex;
+	gap: var(--spacing-sm);
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+/* 
+===================================
+Positions
+===================================
+*/
+
+.positions .wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-med);
+	max-width: 780px;
+	text-align: center;
+	align-items: center;
+	color: var(--text-color-dark);
+}
+/* 
+===================================
+Get Involved
+===================================
+*/
+
+.get-involved {
+	background: linear-gradient(180deg, #6c4681 -5.71%, #184f85 103.44%);
+	color: var(--text-color-light);
+}
+
+/* flex layout */
+.get-involved .wrapper {
```

Comment:
The text and image should be vertically centered: `align-items: center;`

<img width="1438" alt="Screen Shot 2024-01-11 at 3 44 22 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/52edd84d-9f33-42f0-8eb7-a1ae0f233f63">